app/Providers/AppServiceProvider.php

```diff
@@ -28,7 +28,7 @@ public function register()
      */
     public function boot()
     {
-        Model::preventLazyLoading();
+        Model::shouldBeStrict();
```

Comment:
- Registration route is throwing on user creation. Missing `remember_token` on fillables.